src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,115 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int GROW_FACTOR = 2;
+    private int threshold;
```

Comment:
current threshold can be calculated based on size and array length, not need to have separate variables for them
```suggestion
```

Reply:
I don`t agree that this variable is not needed, because I actively use it in the resizeIfNeeded method. Without this variable, this method will be unreadable